rust/sbp/src/json/ser.rs

```diff
@@ -19,7 +20,7 @@ const BASE64_BUFLEN: usize = BUFLEN * 4;
 pub fn to_writer<W, M>(mut writer: W, msg: &M) -> Result<(), JsonError>
 where
     W: io::Write,
-    M: SbpMessage + Serialize,
+    M: SbpMessage + Serialize + WireFormat + Clone,
```

Comment:
I think WireFormat was introduced in all of these methods because it was removed form SbpMessage?  Are we sure that all of these methods really need WireFormat?

Reply:
The call to `crate::ser::to_buffer` on line 239 requires `WireFormat`, which then has to be propagated up to pretty much all of the functions in `json::ser`. I took out `WireFormat` from the `SbpMessage` trait in an abandoned refactoring, so if you prefer i can re-add it.

Finally regarding benchmarks, it seems like the rust version is too fast and I'll need to bump some of the expected values.

Reply:
> I took out WireFormat from the SbpMessage trait in an abandoned refactoring, so if you prefer i can re-add it.

Let's re-add it if it's no longer needed